changelog/8249.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix ``_pytest.faulthandler`` for occasions when running with ``twisted.logger`` and using ``pytest --capture=no``.
```

Comment:
```suggestion
Fix the ``faulthandler`` plugin for occasions when running with ``twisted.logger`` and using ``pytest --capture=no``.
```

Reply:
Thanks. I've amended to changelog item like you suggested.